experiments/transfer_learn_benchmark.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+# Train PPO2 experts using reward models from experiments/imit_benchmark.sh
+
+TIMESTAMP=$(date --iso-8601=seconds)
+REWARD_MODELS_DIR=reward_models
+LOG_ROOT="output/train_experts/${TIMESTAMP}"
+RESULTS_FILE="results.txt"
+ALGORITHM="gail"
+extra_configs=""
+
+
+SEEDS="0 1 2"
+ENVS="cartpole mountain_car acrobot half_cheetah "
+ENVS+="hopper walker ant humanoid reacher "
+
+
+TEMP=$(getopt -o f -l fast,gail,airl,run_name:,log_root: -- $@)
+if [[ $? != 0 ]]; then exit 1; fi
+eval set -- "$TEMP"
+
+while true; do
+  case "$1" in
+    # Fast mode (debug)
+    -f | --fast)
+      ENVS="cartpole mountain_car"
+      SEEDS="0"
+      extra_configs+="fast "
+      shift
+      ;;
+    --gail)
+      ALGORITHM="gail"
+      shift
+      ;;
+    --airl)
+      ALGORITHM="airl"
+      shift
+      ;;
+    --run_name)
+      extra_options+="--name $2 "
```

Comment:
Where is this variable used?

Reply:
This is used for grouping TF runs and maybe analysis tables in the future (more specifically, the sacred utils directory finder can filter on run name). Adding a comment for this.

Reply:
(also adding extra_options to the parallel command so that it is actually used)